app/assets/stylesheets/landing_page/mixins/_buttons.scss

```diff
@@ -0,0 +1,31 @@
+
+/*
+Generic landing page button
+
+You may want to override following properties:
+
+- background-color
+- border-color
+- height
+
+*/
+.landing-page__button {
+  $button-border-size: 2px;
+
+  font-family: ProximaNovaSoft-Bold;
+  font-size: 16px;
+  letter-spacing: 0px;
+  padding: (16px - $button-border-size) (32px - $button-border-size);
+  display: inline-block;
+
+  border-radius: 1000px; // Is there cleaner way to do this?
```

Comment:
This is how I did it as well #toisaalla to get full roundness, you can just comment "high enough to get fully rounded button" etc.